AUTHORS.md

```diff
@@ -1,7 +1,22 @@
-# Contributors to OpenCoarrays
+Contributors to OpenCoarrays
+============================
+
+[![Download as PDF][pdf img]](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/AUTHORS.pdf)
+
+Download this file as a PDF document
+[here](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/AUTHORS.pdf).
```

Comment:
Like this! In other news, I am still mastering the art of the merge commit (apparently) as this PR is listed as having conflicts. Anyway, I know how to resolve them by hand so that's what I'll do...